src/app/core/i18n/en-US.ts

```diff
@@ -1,13 +1,13 @@
 export const ENUS = {
-  slogan: 'A based on Antd for web applications',
+  slogan: 'A front-end solution based on Antd',
   'app.header.search': 'Search in ng-alain.com',
   'app.home.preview': 'Preview',
-  'app.home.start': 'Start',
-  'app.home.slogan': 'Out-of-box UI solution for enterprise applications, Let developers focus on business.',
+  'app.home.start': 'Get started',
+  'app.home.slogan': 'Out of box UI solution based on Antd for enterprise applications, providing more common modules and let developers focus on business.',
```

Comment:
```suggestion
  'app.home.slogan': 'Out-of-box UI solution based on Antd for enterprise applications, providing more common modules and let developers focus on business.',
```

`Out-of-box` 应该是合理的，因为所有跟开箱即用有关的单词都这么写，我想应该是没有错。